openmp/libomptarget/include/OpenMP/InteropAPI.h

```diff
@@ -24,9 +25,7 @@
 #endif
 #endif
 
-#ifdef __cplusplus
 extern "C" {
-#endif
 
 /// TODO: Include the `omp.h` of the current build
 /* OpenMP 5.1 interop */
```

Comment:
I'd move it to `omp.h` as well, since it is also "OpenMP" type. In this way, we don't need to have this header.